### ISSUE TITLE
bug: ormnomnom expects primary key to be generated by db

### ISSUE THREAD

**Issue by ashleygwilliams** [chrisdickinson/ormnomnom#24]
for example, given the following table:

```
var dbm = require('db-migrate')
var type = dbm.dataType

exports.up = function (db, callback) {
  db.runSql("DROP TYPE IF EXISTS banned_email_type; CREATE TYPE banned_email_type AS ENUM ('spam', 'malicious', 'other');", function () {
    db.createTable('banned_email', {
      columns: {
        email: { type: type.STRING, primaryKey: true },
        type: { type: 'banned_email_type' },
        note: { type: type.TEXT },
        created: { type: type.DATE_TIME },
        updated: { type: type.DATE_TIME },
        deleted: { type: type.DATE_TIME }
      }
    }, function () {
      db.runSql('CREATE UNIQUE INDEX banned_email_idx ON banned_email (email) WHERE deleted IS NULL', function () {
        callback()
      })
    })
  })
}

exports.down = function (db, callback) {}
```

and model:

```
BannedEmail.objects = orm(BannedEmail, {
  email: joi.string().required(),
  type: joi.string().only([
    'spam',
    'malicious',
    'other'
  ]).required(),
  note: joi.string().required(),
  created: joi.date().default(() => new Date(), 'current date'),
  updated: joi.date().default(() => new Date(), 'current date'),
  deleted: joi.date().default(null)
}, {
  tableName: 'banned_email',
  primaryKey: 'email'
})
```

ornnomnom throws the following error:

```
request { email: 'evilspam@bot.me',
  type: 'spam',
  note: '1800 technical support' }
error: null value in column "email" violates not-null constraint
```

even though all required values are being passed correctly

**Comment by ashleygwilliams** [comment]
this is actually fixable if you simply do not define the `primary key` in the model... so perhaps this is a docs sitch over a code fix ;)

**Comment by nlf** [comment]
this was a bug where generated queries were always dropping the primary key during inserts, it has been fixed in 5.0.1 and should work as expected
